lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -314,4 +315,33 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         });
     return Container();
   }
+
+  void settingModalBottomSheet(BuildContext context, Document document) {
+    showModalBottomSheet(
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(10.0),
+          topRight: Radius.circular(10.0)
+        )),
+      context: context,
+      builder: (BuildContext buildContext) {
+        return Container(
+          child: Wrap(
+            children: <Widget>[
+              ListTile(
+                leading: SvgPicture.asset(
+                  imagePath.icFileDownload,
+                  width: 30,
+                  height: 30,
+                  fit: BoxFit.fill),
+                title: Text('Download to device', style:
+                    TextStyle(fontSize: 16, color: AppColor.documentNameItemTextColor)),
+                onTap: () => mySpaceViewModel.downloadFile(document.documentId)
```

Comment:
this context menu will be closed whenever tap on Download?

Reply:
Yes, it's the expected flow isn't it ?